src/components/IconButton/IconButton.tsx

```diff
@@ -4,7 +4,7 @@ import classNames from 'classnames';
 import styles from './IconButton.module.scss';
 import { IconProps } from '../Icons/Icons.types';
 
-export interface Properties {
+export interface IconButtonProperties {
```

Comment:
Made this change because I discovered the generic `Properties` can't be used since we export these all as one huge index file and I got name collisions :). I can pull this into a separate PR if you'd prefer.